frappe/utils/error.py

```diff
@@ -233,3 +233,34 @@ def wrapper_raise_error_on_no_output(*args, **kwargs):
 			return response
 		return wrapper_raise_error_on_no_output
 	return decorator_raise_error_on_no_output
+
+def get_app_details_from_stack(skip_frames=0, ignore_files=()):
```

Comment:
```suggestion
def get_app_details_from_stack(skip_frames=1, ignore_files=()):
```

Maybe 1 is better default here. 

some_app -> your_function -> this function. So 2 frames need to be skipped.